compiler/test/dotty/tools/dotc/reporting/TestReporter.scala

```diff
@@ -86,7 +86,7 @@ object TestReporter {
   private[this] def initLog() = if (logWriter eq null) {
     val date = new Date
     val df0 = new SimpleDateFormat("yyyy-MM-dd")
-    val df1 = new SimpleDateFormat("yyyy-MM-dd-HH:mm:ss")
+    val df1 = new SimpleDateFormat("yyyy-MM-dd-HH.mm.ss")
```

Comment:
Not sure `.` is the best separator as it is also used for file's extension

Reply:
I see, is `_` an alternative ?

Reply:
i propose ` "yyyy-MM-dd'T'HHmmss"`

Reply:
:+1: pushed with rebase